tk-demos/widtrib/plop.pl

```diff
@@ -19,6 +19,7 @@
 
 require 5.002;
 use strict;
+use warnings;
```

Comment:
this "use warnings" contradicts to "use 5.002";

Ok, I do not want "use warnings;" in tcl/tk while they have this : 

die sprintf "Incorrect use of pragma '%s' at %s line %d.\n", __PACKAGE__, +(caller)[1,2]
    if __FILE__ !~ ( '(?x) \b     '.__PACKAGE__.'  \.pmc? \z' )
    && __FILE__ =~ ( '(?x) \b (?i:'.__PACKAGE__.') \.pmc? \z' );

Reply:
...sorry for being dumb in this regard.....